core/tests/puppeteer-acceptance-tests/utilities/common/test-constants.ts

```diff
@@ -169,6 +169,15 @@ export default {
       __dirname,
       '../../data/classroom-banner.png'
     ),
+    IntroContentHiVoiceover: path.resolve(
```

Comment:
I think the names are a bit confusing. Maybe something like "IntroContentVoiceoverInHindi" is better. What do you think? Same with the mp3 files.

Reply:
Updated according to your suggestion. 
Thanks!